### PR TITLE
docs: Fixed grammatical/typographical errors in the website

### LINKED ISSUE #183: Fix grammatical errors
### Description

Fixed grammatical/typographical errors in the website.

## Definition of Done
- [x] All of the required items are completed.
- [x] Approval by 1 mentor.

## Estimation
30 min

### PR DESCRIPTION
**Pull request by antrikshmisri**
### Description

Fixed #183 grammatical/typographical errors in the website.

### Type of Change: Documentation
**Delete irrelevant options.**

- Code
- Quality Assurance
- User Interface
- Outreach
- Documentation

**Code/Quality Assurance Only**
- Bug fix (non-breaking change which fixes an issue)
- This change requires a documentation update (software upgrade on readme file)
- New feature (non-breaking change which adds functionality pre-approved by mentors)

### Checklist:
**Delete irrelevant options.**

- [✓ ] My PR follows the style guidelines of this project
- [ ✓] I have made corresponding changes to the documentation

**Code/Quality Assurance Only**
- [ ✓] My changes generate no new warnings

### CODE REVIEW

File: src/content/home.json [hunk 1]
```diff
@@ -11,7 +11,7 @@
...
 				},
 				{  
-					"par": "Our code of conduct is simple- 'Be nice, Be respectful' which is the one of the many reasons the community coexists in harmony. The members should contribute actively keeping the policies of being repectful to everyone and refrainging from conflicts, unruly behaviour and speech. Anyone who wishes to join the community is expected to follow these guidelines.\nIf you are subject to or witness unacceptable behavior, or have any other concerns, please notify a community organizer or adminas soon as possible."
+					"par": "Our code of conduct is simple- 'Be nice, Be respectful' which is the one of the many reasons the community coexists in harmony. The members should contribute actively keeping the policies of being respectful to everyone and refraining from conflicts, unruly behaviour and speech. Anyone who wishes to join the community is expected to follow these guidelines.\nIf you are subject to or witness unacceptable behavior, or have any other concerns, please notify a community organizer or admins soon as possible."
```
Comment (devkapilbansal) [inline src/content/home.json:14]:
```suggestion
					"par": "Our code of conduct is simple- 'Be nice, Be respectful' which is the one of the many reasons the community coexists in harmony. The members should contribute actively keeping the policies of being respectful to everyone and refraining from conflicts, unruly behaviour and speech. Anyone who wishes to join the community is expected to follow these guidelines.\nIf you are subject to or witness unacceptable behavior, or have any other concerns, please notify a community organizer or admins as soon as possible."
```

**Review by devkapilbansal — COMMENTED** [review]
LGTM
@antrikshmisri can you please verify that the rest documentation is error free

**Comment by antrikshmisri** [comment]
@devkapilbansal  Sure , i'll cross check

**Comment by antrikshmisri** [comment]
@devkapilbansal I have made all the changes , could you please review and merge .

**Review by devkapilbansal — APPROVED** [review]
LGTM :+1:

**Comment by nandini45** [comment]
@antrikshmisri  which issue is this? 
@devkapilbansal  do u know which issue is the person working on?

**Review by annabauza — APPROVED** [review]

**Comment by annabauza** [comment]
@antrikshmisri I don't see linked issue it solves. The way we work here is issue need to exist and assigned to you. @nandini45 can you rise the issue and assign it to @antrikshmisri ?

**Comment by antrikshmisri** [comment]
@annabauza Oh my bad , should i create an issue and link it to this PR?

**Comment by nandini45** [comment]
Sure i will do it by tomorrow

**Comment by annabauza** [comment]
@antrikshmisri no. Nandini will do that. You should not assign yourself to the task, someone else should do it. Thanks.

**Pull request merged**